### ISSUE TITLE
white-space: pre instead of &nbsp;

### ISSUE THREAD

**Issue by syranide** [ritzyed/ritzy#10]
Is there a good reason you're not using `white-space: pre` instead of replacing all spaces with `&nbsp;`? (`&nbsp;` isn't enough to cover all cases anyway)

**Comment by rocketraman** [comment]
No, not particularly. I think it makes sense and would actually allow me to strip out some of the logic for replacing spaces with `&nbsp` (which, as you said, doesn't cover every case anyway).

**Comment by rocketraman** [comment]
I will note that use of `&nbsp;` is a surface rendering detail though -- the underlying content is unaffected.

**Comment by rocketraman** [comment]
I'd like to leave this open for tracking purposes. Thanks.
